server/plugins/LFUGens.cpp

```diff
@@ -897,7 +897,9 @@ void LFGauss_Ctor(LFGauss* unit)
 	}
 	unit->mPhase = -1.0;
 
-	//LFGauss_next_k(unit, 1);
```

Comment:
hmmm so it seems there _was_ an init call in olden times, but it was commented out. any idea why?